Gopkg.lock

```diff
@@ -183,6 +207,12 @@
   revision = "2f5df55504ebc322e4d52d34df6a1f5b503bf26d"
 
 [[projects]]
+  branch = "master"
```

Comment:
This file will likely disappear from the change after I merge with master again, it is now here because at the time Costin's PR was not merged.